charts/camunda-platform-alpha-8.8/templates/core/configmap.yaml

```diff
@@ -133,11 +133,11 @@ data:
               url: {{ include "camundaPlatform.elasticsearchURL" . | quote }}
               index:
                 prefix: {{ .Values.global.elasticsearch.prefix | quote }}
-              {{- if .Values.core.archiver.retention.enabled }}
+              {{- if .Values.core.history.retention.enabled }}
```

Comment:
@hisImminence I would expect we have a different retention config for ES / OS exporter then for the Camunda Exporter, might be good to follow up on this.

Reply:
In the context of ES/ OS exporter only needed for Optimize? (data probably will be store longer than Camunda Exporter, which is responsible only for core components?)

Reply:
Yeah i think in general it should be configurable separately as it is two different things.

Zeebe Es or OS exporter are likely shortlived as it is consumed by another component, the importer.

Optimize has anyway a separate config for there retention, and yes that will be longer.

The data written by Camunda Exporter is the historic data for Operate/Taaklist that will stay longer as the Zeebe Es Exporter data but probably not that long as Optimize.